tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -137,7 +137,7 @@ var apiCreateCmd = &cobra.Command{
 
 var apiUpdateCmd = &cobra.Command{
     Use:           "update API_PATH API_VERB ACTION",
-    Short:         wski18n.T("update an existing API"),
+    Short:         wski18n.T("update an existing API, or create a new API if it does not exist"),
```

Comment:
The `wsk api update` command is not yet implemented; however, the current plan is for it to only update, not create an API.